infra/aws/terraform/modules/eks-prow-iam/policy_apply.tf

```diff
@@ -90,6 +101,12 @@ data "aws_iam_policy_document" "eks_apply" {
       "logs:PutRetentionPolicy",
       "logs:TagLogGroup",
       "s3:PutObject",
+      "sqs:createqueue",
```

Comment:
Are these permissions required by prow, or by Karpenter? Likewise fir the Eventbridge API permissions

Reply:
These are required by the karpenter module, it creates a SQS queue for the event messaging.